src/ebay-button/button-expand.tsx

```diff
@@ -1,14 +1,20 @@
-import React, { FC } from 'react'
+import React, { FC, ReactNode } from 'react'
 import EbayButtonCell from './button-cell'
 import EbayButtonText from './button-text'
 import { EbayIcon } from '../ebay-icon'
 
-const EbayButtonExpand: FC = ({ children }) => (
+type Props = {
+    className?: string;
+    children: ReactNode;
+    showDropdown?: boolean;
+}
```

Comment:
@don-siu Could you synch your feature branch with main one? I think your feature branch is out dated with the main one.

Also, you don't need to redefine children on your `Props` here since `FC` will provide type for children.

Reply:
cool @abiyasa , I will pull upstream in the coming batch of changes. 

will update for the `Props` issue

Reply:
hmm I have to define props to pass `showDropDown`. What do you mean by FC will provide type of children ?

Reply:
@don-siu  You need to remove only the `children` from your Props. FC will provide that for you (see https://github.com/facebook/create-react-app/pull/8177).

```tsx
type Props = {
    className?: string;
    showDropdown?: boolean;
}
```

Reply:
updated